requirements/test.txt

```diff
@@ -9,22 +9,27 @@ appdirs==1.4.4
     # via fs
 asgiref==3.7.2
     # via django
+attrs==23.1.0
+    # via
+    #   outcome
+    #   trio
 backports-zoneinfo[tzdata]==0.2.1
     # via
     #   celery
     #   kombu
     # via celery
-bok-choy==2.0.2
-    # via -r requirements/test.in
     # via
     #   -c requirements/constraints.txt
     #   event-tracking
 certifi==2023.7.22
-    # via requests
+    # via
+    #   requests
+    #   selenium
```

Comment:
It doesn't look like anything else is using selenium either, so it's probably safe to remove that from `test.in` as well.

Reply:
@feanil should i remove the selenium under the scope of this PR? or create separate one?

Reply:
I think we should remove selenium along with bok-choy if it's not being used independently of bok-choy.

Reply:
@feanil done.